src/main/java/com/example/notes/controller/NoteController.java

```diff
@@ -0,0 +1,25 @@
+package com.example.notes.controller;
+
+import com.example.notes.dto.NoteDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/note")
+@RequiredArgsConstructor
+public class NoteController {
```

Comment:
In the controller folder add a note-controller.request.http file with a test GET request (to enable easy testing from the IDE)